src/client/search/SearchActions.js

```diff
@@ -22,3 +25,24 @@ const addQueryParamTypes = (searchQuery, query, paramTypes) => {
     paramTypes,
   };
 };
+
+export const INVALIDATE_FEED_RESULTS = 'INVALIDATE_FEED_RESULTS';
+export const invalidateFeedResults = () => {
+  return (dispatch, getState) => {
+    dispatch({ 'type': INVALIDATE_FEED_RESULTS });
+    dispatch(searchApiForFeed(getState().searchTerms));
+  };
+};
+
+export const UPDATE_FEED_RESULTS = 'UPDATE_FEED_RESULTS';
```

Comment:
Can we change this to REQUEST_FEED_RESULTS to not confuse it with other UPDATE actions that will be implemented so we know what are request actions and what aren't ?